jekyll/_cci2/about-circleci.md

```diff
@@ -1,56 +1,23 @@
 ---
 layout: classic-docs
-title: "About CircleCI"
-short-title: "About CircleCI"
+title: "Overview"
+short-title: "Overview"
 description: "Starting point for CircleCI 2.0 docs"
 categories: [getting-started]
 order: 1
 ---
 
-CircleCI is a modern **continuous integration** and **continuous delivery** platform.
+CircleCI is a modern continuous integration and continuous delivery (CI/CD) platform. The hosted solution is available at <https://circleci.com/>. The [CircleCI Enterprise](https://circleci.com/enterprise/) solution is installable inside your private cloud or data center. Both are free to try for two weeks. CircleCI automates build, test, and deployment of software artifacts to any service with an API or to any machine with `sshd`. 
```

Comment:
The end of the last sentence above:

> CircleCI automates build, test, and deployment of software artifacts to any service with an API or to any machine with `sshd`. 

is confusing and not accurate. We should avoid saying 'artifacts' here since they are a separate aspect of the process. It's not necessary (or accurate) to say API or sshd.

Suggest simplifying to:

> CircleCI automates build, test, and deployment of software.